### ISSUE TITLE
Application messages are not announced

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3942]
**Issue Description:** The messages from the application about issues with the LLM chat are not being announced for assistive technology users. This lack of announcement prevents users relying on screen readers or braille displays from receiving important notifications. The expected behavior is that when a message is posted in the chat thread assistive technology users will also be notified of the contents of the message.

WCAG Criteria:
[1.3.1 Info and Relationships - Level A](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

Screenshot or Screen Recording:
[Screen Reader Review](https://share.zight.com/rRuA9w5Q)

**Notes**:
Ensure all updates to the chat thread are included in the live region update process to notify assistive technology users (e.g., screen readers, braille displays) of message status.

**Comment by Tanvez** [comment]
@danny-avila I can look into this if you haven't started working on it yet.
